Source/CombatExtended/CombatExtended/Comps/CompAmmoUser.cs

```diff
@@ -295,6 +296,10 @@ public void TryStartReload()
                 return;
             }
 
+            // R&G compatibility, prevents an initial attempt to reload while moving
+            if (Wielder.stances.curStance.GetType() == rgStance)
```

Comment:
What exactly is the purpose behind rgStance, to check if the shooter is currently shooting while moving? If so it might be better to simply check current movement speed, which is how a number of other Run and Gun/GiddyUp compatibility features are done.

Reply:
Just testing for the pawn being in motion blocks manually issued reloads while moving, sadly.

Reply:
Don't manually ordered reloads stop the pawn already? I imagine this can be circumvented by applying the check to automatically triggered reloads only or else by stopping the pawn as part of the gizmo.

Reply:
Conversation resolved via Discord